csvAddTeam.py

```diff
@@ -0,0 +1,55 @@
+from flask import render_template, request
+from flask.views import MethodView
+from flask_cas import login_required
+from sqlalchemy.exc import SQLAlchemyError
+import gbmodel
+import datetime
+import csv
+import io
+
+
+class AddTeamCSV(MethodView):
+    @login_required
+    def get(self):
+        return render_template('csvAddTeam.html')
+
+    def post(self):
+        session = gbmodel.capstone_session()
+        teams_table = gbmodel.teams()  # Accessor to the teams table
+        students_table = gbmodel.students()  # Accessor to the students table
+        file = request.files['teamcsv']
+        stream = io.StringIO(file.stream.read().decode("UTF8"), newline=None)
+        csv_reader = csv.reader(stream, delimiter=',')
+        for row in csv_reader:
+            student_name = row[0]
+            student_id = row[1]
+            team_name = row[2]
+
+            # Get date to determine session ID
+            current_date = datetime.datetime.now()
+            month = int(current_date.month)
+            year = current_date.year
+            if month in range(9, 11):
+                term = "Fall"
+            elif month in range(3, 5):
+                term = "Spring"
+            elif month in range(6, 8):
+                term = "Summer"
+            else:
+                term = "Winter"
```

Comment:
We might consider moving this date calculation code out of the for loop (if we end up keeping it--see the comment below). The same session_id is getting re-calculated for every student, which may be fine, but it's extra CPU work we don't need.

Reply:
This is something that can be fixed during code cleanup? Or should it be done before merging?